docs/proposals/15-manage-bastions-and-ssh-key-pair-rotation.md

```diff
@@ -221,8 +221,8 @@ Currently, the `ssh` key pair for the shoot nodes are created once during shoot
         - Generates new `ssh-keypair` secret.
         - The `OperatingSystemConfig` needs to be re-generated and deployed with the new and old `ssh` public key.
     - As usual (for more details, see [Contract: OperatingSystemConfig Resource](../extensions/operatingsystemconfig.md)):
-        - Once the `cloud-config-<X>` secret in the `kube-system` namespace of the shoot cluster is updated, it will be picked up by the [`downloader` script](https://github.com/gardener/gardener/blob/master/pkg/component/extensions/operatingsystemconfig/downloader/templates/scripts/download-cloud-config.tpl.sh) (checks every 30s for updates).
-        - The `downloader` runs the ["execution" script](https://github.com/gardener/gardener/blob/master/pkg/component/extensions/operatingsystemconfig/executor/templates/scripts/execute-cloud-config.tpl.sh) from the `cloud-config-<X>` secret.
+        - Once the `cloud-config-<X>` secret in the `kube-system` namespace of the shoot cluster is updated, it will be picked up by the [`downloader` script](../pkg/component/extensions/operatingsystemconfig/downloader/templates/scripts/download-cloud-config.tpl.sh) (checks every 30s for updates).
```

Comment:
```suggestion
        - Once the `cloud-config-<X>` secret in the `kube-system` namespace of the shoot cluster is updated, it will be picked up by the [`downloader` script](../../pkg/component/extensions/operatingsystemconfig/downloader/templates/scripts/download-cloud-config.tpl.sh) (checks every 30s for updates).
```